src/ray/object_manager/object_manager.h

```diff
@@ -361,28 +405,19 @@ class ObjectManager : public ObjectManagerInterface {
   ObjectStoreNotificationManager store_notification_;
   ObjectBufferPool buffer_pool_;
 
-  /// This runs on a thread pool dedicated to sending objects.
-  boost::asio::io_service send_service_;
-  /// This runs on a thread pool dedicated to receiving objects.
-  boost::asio::io_service receive_service_;
+  /// Multi-thread asio service, deal with all send and receive request
+  boost::asio::io_service rpc_service_;
 
   /// Weak reference to main service. We ensure this object is destroyed before
   /// main_service_ is stopped.
   boost::asio::io_service *main_service_;
 
-  /// Used to create "work" for send_service_.
-  /// Without this, if send_service_ has no more sends to process, it will stop.
-  boost::asio::io_service::work send_work_;
-  /// Used to create "work" for receive_service_.
-  /// Without this, if receive_service_ has no more receives to process, it will stop.
-  boost::asio::io_service::work receive_work_;
+  boost::asio::io_service::work rpc_work_;
 
-  /// Runs the send service, which handle
-  /// all outgoing object transfers.
-  std::vector<std::thread> send_threads_;
-  /// Runs the receive service, which handle
-  /// all incoming object transfers.
-  std::vector<std::thread> receive_threads_;
+  /// Runs the rpc service, which handle all send request and receive request
```

Comment:
```suggestion
  /// The thread pool used for running `rpc_service_`. 
```
`rpc_service_`'s documentation already explain what it does.